jolt-core/src/jolt/instruction/virtual_assert_eq_signs.rs

```diff
@@ -0,0 +1,88 @@
+use crate::poly::field::JoltField;
+use rand::prelude::StdRng;
+use rand::RngCore;
+use serde::{Deserialize, Serialize};
+
+use super::{JoltInstruction, SubtableIndices};
+use crate::{
+    jolt::subtable::{eq_msb::EqMSBSubtable, LassoSubtable},
+    utils::instruction_utils::chunk_and_concatenate_operands,
+};
+
+#[derive(Copy, Clone, Default, Debug, Serialize, Deserialize)]
+pub struct ASSERTEQSIGNSInstruction(pub u64, pub u64);
+
+impl JoltInstruction for ASSERTEQSIGNSInstruction {
+    fn operands(&self) -> (u64, u64) {
+        (self.0, self.1)
+    }
+
+    fn combine_lookups<F: JoltField>(&self, vals: &[F], _: usize, _: usize) -> F {
+        vals[0]
+    }
+
+    fn g_poly_degree(&self, _: usize) -> usize {
+        1
+    }
+
+    fn subtables<F: JoltField>(
+        &self,
+        _: usize,
+        _: usize,
+    ) -> Vec<(Box<dyn LassoSubtable<F>>, SubtableIndices)> {
+        vec![(Box::new(EqMSBSubtable::new()), SubtableIndices::from(0))]
+    }
+
+    fn to_indices(&self, C: usize, log_M: usize) -> Vec<usize> {
+        chunk_and_concatenate_operands(self.0, self.1, C, log_M)
+    }
+
+    fn lookup_entry(&self) -> u64 {
+        ((self.0 as i32).signum() == (self.1 as i32).signum()
+            || (self.0 as i32).signum() >= 0 && (self.1 as i32).signum() >= 0)
```

Comment:
I think the first clause is unnecessary?
```suggestion
        (self.0 as i32).signum() >= 0 && (self.1 as i32).signum() >= 0)
```

Reply:
was trying to check the case where both signs are negative. made it more explicit https://github.com/a16z/jolt/pull/370/commits/90f5cb5b9f88423ea096e12ebbef45d129c56f79